docs/proposals/job.md

```diff
@@ -0,0 +1,145 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller - which will be responsible
+for managing pod(s) that require to run-once to a completion, in contrast to what
```

Comment:
side note - FWIW you're eventually going to need the inverse forgiveness for batch.  Some folks call it claim-lease-lifetime, in order to determine when to reschedule.